portfolio/src/main/webapp/index.html

```diff
@@ -35,11 +49,18 @@ <h1>JENNIFER JIMENEZ</h1>
         like to paint (I'm bad at it) and play guitar, though I don't do it as much as I would like to. During quarantine
         I got into skateboarding a little bit and I would like to continue during the summer. :)
       </p>
-      <img src="/images/jen_nyc.jpg" id="profile_pic"/>
-      <h2>Get to know me more!</h2>
-        <p>Click here to learn a random fact about me:</p>
+
+  <h2>Get to know me more!</h2>
+    <p>Click here to learn a random fact about me:</p>
+    <!-- TODO:  Query to JS file -->
       <button onclick="addRandomGreeting()">Jen Fact!</button>
-      <div id="greeting-container"></div>   
+      <div id="greeting-container"></div> 
```

Comment:
nit: "fact-container" is probably more descriptive here now. The method above could probably be something like `addRandomFact()` too.